### DISCUSSION TITLE
Reference for C4

### DISCUSSION THREAD

**Discussion by neborkia** [svsdval/video2midi#44] [Ideas]
«EDIT» Nevermind, just discovered the "Base Octave" parameter to adjust to the first C in the video :)

First of all, thanks for this wonderful app.
I'm doing my first tests and I'm having difficulty positioning the keys in case the video shows a reduced keyboard and not the classic with 88 keys.
Maybe I'm the one who can't activate it, but it would be really useful to have a reference for the C4 to facilitate its positioning.

**Comment by svsdval** [comment]
I can highlight the c4 key / base octave if it helps

**Reply by binokaryg** [reply]
Yes, please!

**Reply by svsdval** [reply]
Highlighted with red underline ;)
